src/ict/metadata/objects.py

```diff
@@ -0,0 +1,182 @@
+"""Metadata Model."""
+import re
+from functools import singledispatchmethod
+from pathlib import Path
+from typing import Any, Optional, Union
+
+from pydantic import (
+    AnyHttpUrl,
+    BaseModel,
+    EmailStr,
+    Field,
+    RootModel,
+    WithJsonSchema,
+    field_validator,
+    model_validator,
+)
+from typing_extensions import Annotated
+
+from ict.version import Version
+
+
+class Author(RootModel):
+    """Author object."""
+
+    root: str
+
+    @field_validator("root")
+    @classmethod
+    def check_author(cls, value):
+        """Check the author follows the correct format."""
+        if not len(value.split(" ")) == 2:
```

Comment:
I think you need to remove this condition, as has been previously mentioned. In many cultures, it is indeed common to have more than two components in a full name, e.g. Chinese and Korean.

Reply:
Also, some authors may insist in prefixes and suffixes.